client/post-editor/editor-sharing/publicize-services.jsx

```diff
@@ -1,59 +1,62 @@
 /**
  * External dependencies
  */
-var React = require( 'react' );
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { map, reduce } from 'lodash';
 
 /**
  * Internal dependencies
  */
-var serviceConnections = require( 'my-sites/sharing/connections/service-connections' ),
-	EditorSharingPublicizeConnection = require( './publicize-connection' );
+import EditorSharingPublicizeConnection from './publicize-connection';
+import { getCurrentUserId } from 'state/current-user/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteUserConnections } from 'state/sharing/publicize/selectors';
+
+class EditorSharingPublicizeServices extends Component {
+	static propTypes = {
+		post: PropTypes.object,
+		connections: PropTypes.array.isRequired,
+		newConnectionPopup: PropTypes.func.isRequired
+	};
+
+	render() {
+		const services = reduce( this.props.connections, ( memo, connection ) => {
+			if ( connection.label in memo ) {
```

Comment:
WordPress' "There's a plugin for that!" is your "There's a lodash function for that!" 😀

Yes, I want `groupBy` haha